### PR TITLE
Jdbc

### PR DESCRIPTION
**Pull request by JYCabello**
Removed the ORM from the project and made the migrations explicit with flyway. The students came to this right after being told not to heavily rely on frameworks and ORMs and found it off putting.

### CODE REVIEW

File: simpleFrontEnd/src/modules/Main.js [hunk 1]
```diff
@@ -23,19 +23,20 @@ function Main() {
...
+                {users.map(u => <User user={u}/>)}
             </header>
         </div>
     )
 }
 
-export { Main }
\ No newline at end of file
+function User(props) {
+    return <>
```
Comment (simonbesteiro) [inline simpleFrontEnd/src/modules/Main.js:33]: this empty element is not needed

**Review by danihra-codurance — APPROVED** [review]
Looks good to me

File: Makefile [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+.PHONY: dockerLocal
+dockerLocal: db build_backend_docker build_frontend_docker up
+
 .PHONY: backend
 backend: build_backend_docker push_backend
 
 .PHONY: frontend
 frontend: build_frontend_docker push_frontend
 
+
```
Comment (simonbesteiro) [inline Makefile:13]: this whitespace can be removed

File: simpleFrontEnd/src/modules/Main.js (hunk 1 above)
Comment (JYCabello) [reply simpleFrontEnd/src/modules/Main.js]: Fixed

**Review by simonbesteiro — APPROVED** [review]

**Review by simonbesteiro — APPROVED** [review]

**Pull request merged**